include/lsst/meas/base/pybind11.h

```diff
@@ -0,0 +1,156 @@
+// -*- lsst-c++ -*-
+/*
+ * LSST Data Management System
+ * See COPYRIGHT file at the top of the source tree.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#ifndef LSST_MEAS_BASE_PYBIND11_H
+#define LSST_MEAS_BASE_PYBIND11_H
+
+#include <string>
+
+#include <pybind11/pybind11.h>
+
+#include "lsst/afw/image/Calib.h"
+#include "lsst/afw/image/Wcs.h"
+#include "lsst/afw/table/fwd.h"
+#include "lsst/afw/table/Schema.h"
+#include "lsst/afw/table/SchemaMapper.h"
+
+namespace lsst {
+namespace meas {
+namespace base {
+namespace {
+    namespace py = pybind11;
```

Comment:
I'll fix the indentation, but [using directives should not be visible from header files](https://developer.lsst.io/coding/cpp_style_guide.html#appendix-on-using-using) -- too much risk of side effects in files that include them.